### PR TITLE
iOS: Fixes #3607: keyboards won't cover last line of note

### LINKED ISSUE #3607: iOS Bug: iPhone 11 - last line of text hidden, unable to scroll down to see
This was introduced in issue #1437 already, but it still happens on the iPhone 11 with iOS 13.6. It is fixed on my iPad Air 2 with iOS 13.6, though.

When creating or editing a note, I cannot edit the last line unless I manually position my cursor at the end.
I cannot view the last line until I add another new line, line, which is one of the fixes described in #1437.

## Environment

Joplin version: 10.0.52
Platform: iOS
OS specifics: iPhone 11, iOS 13.6

## Steps to reproduce

1. Create or edit a note.
2. Type a bunch of lines of text.
3. Eventually, when the whole screen is filled with text, the last line of text on the note will go behind the keyboard.

Even if I scroll down to the bottom, I cannot scroll any further to see the last line of text until I add another new line.

Disabling "Predictive Text" on the iOS keyboard does not fix this issue.

## Describe what you expected to happen
I am able to see the last line of text without having to enter a new line at the end of the note.

### Bug on iPhone 11
With predictive text:
298155C1-BDA5-4B58-B8CC-7F55B37B94A1

Without predictive text:
B793CF57-2966-4522-AC64-9DBD6FCDB13E

### Working fine on iPad Air 2

With predictive text:
B21F1560-6A03-4C06-ADE8-3D9688EB3C69

Without predictive text:
37F53081-25C0-4E39-8698-606EC40674CA

### PR DESCRIPTION
**Pull request by innkuika**
This PR doesn't have test units. It's a one-line-code UI bug fix that prevents keyboard on iOS (iPhone in particular) from covering the last line of note.

I did the following manual test on iPhone 12, iPhone 11 Pro Max, iPhone SE:
1. Create a new note
2. Type multiple lines of text (enough to fill one page)
3. Try to scroll to the bottom of the note when editing, making sure that the last line is visible & editable

### CODE REVIEW

File: packages/app-mobile/components/screens/Note.tsx [hunk 1]
```diff
@@ -1141,6 +1141,8 @@ class NoteScreenComponent extends BaseScreenComponent {
 					keyboardAppearance={theme.keyboardAppearance}
 					placeholder={_('Add body')}
 					placeholderTextColor={theme.colorFaded}
+					// need some extra padding for iOS so that the keyboard won't cover last line of the note
```
Comment (laurent22) [inline packages/app-mobile/components/screens/Note.tsx:1144]: Could you also add a link to the original issue please?

**Review by laurent22 — COMMENTED** [review]
Thanks for testing on all three devices. I've just left one comment and then we can merge.

File: packages/app-mobile/components/screens/Note.tsx (hunk 1 above)
Comment (innkuika) [reply packages/app-mobile/components/screens/Note.tsx:1144]: Done

**Comment by laurent22** [comment]
All good now, thank you. As this was a relatively small pull request I encourage you to perhaps check if there's another one you would know how to fix.

**Pull request merged**